### PR TITLE
Added bugfix release notes

### PR DESCRIPTION
**Pull request by tvansteenburgh**
Will fill in bundle revision once it's known.

### CODE REVIEW

File: pages/k8s/release-notes.md [hunk 1]
```diff
@@ -14,6 +14,31 @@ toc: False
...
+ - Fixed handling "not found" error message ([Issue](https://github.com/juju-solutions/charm-azure-integrator/pull/20))
+ - Fixed snapd_refresh smashed by subordinate charm ([Issue](https://github.com/charmed-kubernetes/layer-etcd/pull/148))
+ - Fixed making sure cert has proper IP as well as DNS ([Issue](https://github.com/charmed-kubernetes/layer-etcd/pull/149))
+ - Fixed etcd charm stuck on "Requesting tls certificates" ([Issue](https://github.com/charmed-kubernetes/layer-etcd/pull/150))
+ - Fixed cert relation thrashing due to random SAN order ([Issue](https://github.com/charmed-kubernetes/layer-etcd/pull/151))
+ - Fixed contact point for keystone to be public address ([Issue](https://github.com/charmed-kubernetes/charm-kubernetes-master/pull/2))
+ - Fixed cluster tag not being sent to new worker applications ([Issue](https://github.com/charmed-kubernetes/charm-kubernetes-master/pull/4))
+ - Fixed removal of ceph relations causing trouble ([Issue](https://github.com/charmed-kubernetes/charm-kubernetes-master/pull/6))
+ - Fixed stuck at "Waiting for master components to start" after upgrading ([Issue](https://github.com/charmed-kubernetes/charm-kubernetes-master/pull/7))
```
Comment (hyperbolic2346) [inline pages/k8s/release-notes.md]: This was closed and not accepted.

**Comment by evilnick** [comment]
just a note to self that we will need to change the table in install-manual too/

File: pages/k8s/release-notes.md [hunk 2]
```diff
@@ -14,6 +14,30 @@ toc: False
 ---
 
 
+# 1.14 Bugfix release
+
+### April 23rd, 2019 - [canonical-kubernetes-???](https://api.jujucharms.com/charmstore/v5/canonical-kubernetes-???/archive/bundle.yaml)
```
Comment (kwmonroe) [inline pages/k8s/release-notes.md]: Canonical or Charmed?

Comment (Cynerva) [reply pages/k8s/release-notes.md]: I'll suggest a middle ground. Charmonical.

File: pages/k8s/release-notes.md (hunk 1 above)
Comment (tvansteenburgh) [reply pages/k8s/release-notes.md]: Thanks, removed.

File: pages/k8s/release-notes.md (hunk 2 above)
Comment (tvansteenburgh) [reply pages/k8s/release-notes.md]: Thanks, changed to 'charmed'.

**Pull request merged**